lib/morse_translator.rb

```diff
@@ -40,4 +41,21 @@ def initialize
                     " " => " "}
   end
 
+  def translate(message)
+    translate = ""
+    message.each_char do |strings|
+      translate += dictionary[strings.downcase]
+    end
+    translate
+  end
+
+  def morse_to_eng(message)
```

Comment:
SOOO close!  You just need to take into account the double spaces between words!